client/src/app/install/install-step-3-database.js

```diff
@@ -26,6 +26,7 @@ class InstallStep3Database extends React.Component {
                 {this.renderMessage()}
                 <Form loading={this.state.loading} onSubmit={this.onSubmit.bind(this)}>
                     <FormField name="dbHost" label={i18n('DATABASE_HOST')} fieldProps={{size: 'large'}} required/>
+                    <FormField name="dbPort" label={i18n('DATABASE_PORT')} fieldProps={{size: 'large'}} infoMessage={i18n('DEFAULT_PORT')} required/>
```

Comment:
The last "required" should not be there, according to the 'DEFAULT_PORT' infoMessage.